auraed/src/cells/executable.rs

```diff
@@ -0,0 +1,26 @@
+use cgroups_rs::CgroupPid;
+use log::info;
+use std::io;
+use std::process::{Child, Command, ExitStatus};
+
+#[derive(Debug)]
+pub(crate) struct Executable(Child);
+
+impl Executable {
+    pub fn spawn(mut command: Command) -> io::Result<Self> {
```

Comment:
nit: i still think it's weird that there's no args passed through separately :)

Reply:
I think you were 100% right on not having it in validation. Currently it's in `CellService`. I'm inclined to continue with your suggestion to keep passing it down, especially since I forgot to pass description.